components/contribution-flow/StepDetails.js

```diff
@@ -50,7 +52,7 @@ const StepDetails = ({ onChange, data, collective, tier, showFeesOnTop, router }
 
   return (
     <Box width={1}>
-      {(!tier || tier.amountType === AmountTypes.FLEXIBLE) && (
+      {(!tier || tier.amountType === AmountTypes.FLEXIBLE) && canContributeRecurring(collective, LoggedInUser) && (
```

Comment:
Not from your PR, and we probably want to solve it in a different PR, but this check seems wrong, it should be something like:

```es6
{(!tier || tier.inteval === 'flexible') && ...
```

Reply:
@Betree : Fixed at https://github.com/opencollective/opencollective-frontend/pull/6426 🐼